src/sc/sidechain.h

```diff
@@ -40,6 +41,9 @@ class ScInfo
     // creation data
     ScCreationParameters creationData;
 
+    // immature amounts
+    std::map<int, CAmount> mImmatureAmounts;
```

Comment:
It is better to add the description of the meaning of the key and value, like
key is `maturity height`, value is ...

Reply:
Ok, will add it